internal/kibana/agents.go

```diff
@@ -0,0 +1,226 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Jeffail/gabs/v2"
+	log "github.com/sirupsen/logrus"
+)
+
+// UnEnrollAgent unenrolls agent from fleet
+func (c *Client) UnEnrollAgent(hostname string, force bool) error {
```

Comment:
NIT: could you reorder the methods alphabetically when possible? That would help when searching through the file

Reply:
I think that once this is done, we can merge this PR 😉